ui/lib/core/addon/helpers/transition-to.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Helper from '@ember/component/helper';
+import { getOwner } from '@ember/application';
+
+/*
+template helper that replaces ember-router-helpers https://github.com/rwjblue/ember-router-helpers
+example:
+<MyForm @onSave={{transition-to "vault.cluster.some.route.item" "item-id"}} />
+<MyForm @onSave={{transition-to "vault.cluster.some.external.route"  external=true}} />
+*/
+export default class TransitionTo extends Helper {
```

Comment:
Open to other ideas here, this seemed like a nice and simple way to dynamically use the `transition-to` helper without knowing the router ahead of time. It also was nice to remove an dependency 🎉 

For reference, Chelsea had made a specific `engine-transition-to` helper [here](https://github.com/hashicorp/vault/pull/28281/files#diff-dcaf011bd9819197ab45fe8516708c96243752d62bf3efe0233da2091edfd9b5). Which is what I used as a basis for building this component. 

Also there are no instances now where we pass `external=true` but I've copied her test to validate this works as expected.

Reply:
I like this approach a lot. Nice work!